src/components/academies/AcademyContentCard.jsx

```diff
@@ -109,6 +109,14 @@ const AcademyContentCard = ({
     }
     return showAllOcmCourses;
   };
+
+  const toggleButtonText = useMemo(() => (showMoreBtnEnabled, contentType, title, contentLength) => {
```

Comment:
It would be better to return the text instead of the function here.

Reply:
@saleem-latif `toggleButtonText` is returning a function because it lacks direct access to the parameters `showMoreBtnEnabled`, `contentType`, `title`, and `contentLength` so it is returning a method that can be called with arguments to generate the desired text.. and it cannot be rendered inside `renderableContent` directly as it is not a React function.
Please let me know if i am missing something?

Reply:
In that case it would be better to use a function and remove 'useMemo'